source/agora/common/Config.d

```diff
@@ -291,6 +294,7 @@ private NodeConfig parseNodeConfig (Node* node, const ref CommandLine cmdln)
 
     string data_dir = get!(string, "node", "data_dir")(cmdln, node);
     auto port = get!(ushort, "node", "port")(cmdln, node);
+    auto validator_cycle = get!(uint, "node", "validator_cycle")(cmdln, node);
```

Comment:
This needs to be optional

Reply:
> This needs to be optional

How about this?
`
auto value = get!(uint, "node", "validator_cycle")(cmdln, node);
auto validator_cycle = (value != 0) ? value : NodeConfig.init.validator_cycle;
`

Reply:
There is `opt` for this

Reply:
Ah, my bad, I just realized that `get` also uses the default value.